kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -363,6 +363,100 @@ paths:
         404:
           description: Not found
 
+  /api/clusters/{clusterName}/topics/analyzes:
+    get:
+      tags:
+        - Topics
+      summary: getAllTopicAnalyzes
+      operationId: getAllTopicAnalyzeStates
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: List of all analyzes
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/TopicAnalyzeState'
+        404:
+          description: Not found
+
+  /api/clusters/{clusterName}/topics/{topicName}/analyze:
+    get:
+      tags:
+        - Topics
+      summary: getTopicAnalyzeState
+      operationId: getTopicAnalyzeState
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: topicName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TopicAnalyzeState'
+        404:
+          description: Not found
+    post:
+      tags:
+        - Topics
+      summary: analyzeTopic
+      operationId: analyzeTopic
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: topicName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        201:
+          description: Analyze started
+        404:
+          description: Not found
+    delete:
+      tags:
+        - Topics
+      summary: analyzeTopic
+      operationId: cancelTopicAnalyze
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: topicName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        201:
+          description: Analyze cancelled
```

Comment:
analyze is a verb, in this case it'd be `analysis`
please also consider renaming other stuff accordingly